### PR TITLE
Typo Fix -> Lesson_13_Build_token_minter

### PR DESCRIPTION
**Pull request by pablinche**
Typo fix

### CODE REVIEW

File: Solana_Core/en/Core_4/Lesson_13_Build_token_minter.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Alright, wow wow wow, we've come a long, let's get back to the NFT Staking Progr
 
 You'll notice a couple of different things here. There's now a 'TS' folder which has everything we previously had in our client project, in the Solana Playground.
 
-One import modification is that in (/<project-name>/src/ts/src/utils/constants.ts), the `PROGRAM_ID` is being read from projects keypair.
+One important modification is that in (/<project-name>/src/ts/src/utils/constants.ts), the `PROGRAM_ID` is being read from projects keypair.
```
Comment (helloitsm3) [inline Solana_Core/en/Core_4/Lesson_13_Build_token_minter.md:7]:
```suggestion
Change (/<project-name>/src/ts/src/utils/constants.ts) to `/<project-name>/src/ts/src/utils/constants.ts`
```

**Comment by helloitsm3** [comment]
You're not suppose to copy the whole text in. You're just suppose to change one part. @pablinche. Now just copy this in.

`One import modification is that in `/<project-name>/src/ts/src/utils/constants.ts`, the `PROGRAM_ID` is being read from projects keypair.`

**Review by pablinche — COMMENTED** [review]
Sorry, is this what you meant?

**Comment by helloitsm3** [comment]
> Sorry, is this what you meant?

Yes, thank you!

**Review by helloitsm3 — APPROVED** [review]

**Comment by helloitsm3** [comment]
@all-contributors please add @pablinche for docs

**Pull request merged**